pkg/snapstore/gcs_snapstore.go

```diff
@@ -39,6 +40,7 @@ const (
 	envStoreCredentials       = "GOOGLE_APPLICATION_CREDENTIALS"
 	envStorageAPIEndpoint     = "GOOGLE_STORAGE_API_ENDPOINT"
 	envSourceStoreCredentials = "SOURCE_GOOGLE_APPLICATION_CREDENTIALS"
+	envEnableGCSEmulator      = "GOOGLE_ENABLE_GCS_EMULATOR"
```

Comment:
Can we rename `GOOGLE_ENABLE_GCS_EMULATOR` to `EMULATOR_ENABLED`? It is more clear and effectively conveys whether the emulator is enabled or not.

Reply:
Thanks a lot for the review @seshachalam-yv. I was using that because I use the same variable in druid [fakegcs PR](https://github.com/gardener/etcd-druid/pull/741/files#:~:text=EnvGoogleEnableGCSEmulator%20%3D%20%22GOOGLE_ENABLE_GCS_EMULATOR%22). And the reason why I used `GOOGLE_ENABLE_GCS_EMULATOR` in druid is because all other variables used for different providers use the similar convention of prefixing with provider-name. [Reference](https://github.com/gardener/etcd-druid/blob/7bd94c48b8f39346c79d45f0546c850be72c12ab/pkg/common/constants.go#L53). Due to these reasons, I think it makes sense to keep this. Also `EMULATOR_ENABLED` may sound like it's enabled by default as it uses the word `ENABLED`.  Wdyt?